recipes/geocat-datafiles/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "geocat-datafiles" %}
+{% set version = "2022.03.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NCAR/{{ name }}/archive/v{{ version }}.tar.gz # For building from a specific tag's tarball
+  sha256: 68aef597dbe4223fff37b11dab5e8720b1047085af58c95ddfa27d86c5fb75c9
+  # git_tag: v{{ version }} # For building from a specific tag
+  # git_rev: main # For building from branch for debugging purposes
+  # git_url: https://github.com/NCAR/{{ name }}.git
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed -vv .
+  # skip: true  # [py<38 or win]
```

Comment:
```suggestion
```

Reply:
These commented out lines are used by our internal pre-release testing, we want to keep the pre-release meta.yaml file as similar to the conda-forge feedstock as possible.

May we keep them?

Reply:
(but yes, you can keep them if you want)